modules/riseBidAdapter.js

```diff
@@ -126,12 +137,22 @@ function buildVideoRequest(bid, bidderRequest) {
  * @returns {Array}
  */
 function getSizes(bid) {
-  if (utils.deepAccess(bid, 'mediaTypes.video.sizes')) {
-    return bid.mediaTypes.video.sizes[0];
-  } else if (Array.isArray(bid.sizes) && bid.sizes.length > 0) {
-    return bid.sizes[0];
+  let playerSize = utils.deepAccess(bid, 'mediaTypes.video.playerSize');
+  let width;
+  let height;
+
+  if (utils.isArray(bid.sizes) && bid.sizes.length === 2 && !utils.isArray(bid.sizes[0])) {
```

Comment:
Adding this is fine but FYI, the ad unit is already checked for valid sizes and it's made sure that it's in the right format. Check [this](https://github.com/prebid/Prebid.js/blob/697abe06a4b63bc64635688a69beda084d2dd7e6/src/prebid.js#L75) piece of code.

Reply:
Removed